.github/workflows/ci.yml

```
@@ -24,30 +24,41 @@
+            ${{ runner.os }}-gradle-
+
+      - name: Make gradlew executable
+        run: chmod +x ./gradlew
 
-      - name: Build, install, and run tests with coverage
-        run: mvn clean install -U -B
+      - name: Build, test, and generate coverage report
+        run: ./gradlew clean build jacocoRootReport
 
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)